package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "selfkey-identity-wallet",
 	"productName": "SelfKey Identity Wallet",
-	"version": "0.3.0-beta",
+	"version": "0.4.0",
```

Comment:
@eduardosmoura can you update the version to 1.0.0-beta? We are still going to release a beta wallet.

Reply:
Yes sure @rodrigopavezi